test/e2e/select.test.js

```diff
@@ -142,6 +141,49 @@ describe('vl-select', async () => {
     await assert.eventually.isTrue(select.hasValue('two'));
   });
 
+  it('als gebruiker kan ik een select dynamisch activeren met gegroepeerde opties', async () => {
+    const select = await vlSelectPage.getDynamicGroupingSelect();
+    await assert.eventually.isFalse(select.isDressed());
+    assert.equal((await select.groups()).length, 0);
+
+    await vlSelectPage.activeerDynamischeDataGrouping();
+
+    const selectMetData = await vlSelectPage.getDynamicGroupingSelect();
+    await selectMetData.open();
+    await assert.eventually.isTrue(selectMetData.isDressed());
+
+    const groups = await selectMetData.groups();
+    assert.equal(groups.length, 2);
+    await assert.eventually.equal(groups[0].getLabel(), 'Group 1');
+    await assert.equal(groups[0].options.length, 2);
+    await assert.eventually.equal(groups[0].options[0].getLabel(), 'Label one');
+    await assert.eventually.equal(groups[0].options[0].getValue(), 'one');
+
+    await assert.eventually.equal(groups[1].getLabel(), 'Group 2');
+    assert.equal(groups[1].options.length, 2);
```

Comment:
Is het nodig hier te controleren wat het label is? Op zich is de code al gedekt door group 1 te controleren. Kan geen kwaad om het nog eens te controleren, maar dan zou ik ook option 0 opnieuw controleren zoals hierboven?

Reply:
Ik heb in beide testen de labels van alle groepen + alle items afgetest. 
Daaruit blijkt dat de bij de undressed variant ze gerenderd worden in volgorde van voorkomen in de HTML, en bij de undressed variant dat die zijn choices 'by label' sorteert. Zo aangevuld en van commentaar voorzien in de bijhorende testen.
Gedrag van dressed variant is bij choices.js te beinvloeden met de "shouldSort" property, maar voorlopig is het OK voor ons om 'by label' te sorteren.